main/index.ts

```diff
@@ -102,6 +102,11 @@ const checkForUpdates = () => {
     app.setAsDefaultProtocolClient('kap');
   }
 
+  protocol.registerFileProtocol('file', (request, callback) => {
```

Comment:
Can you add a code comment about why this exists. Might be worth moving it into a util file.

Reply:
Moved to a util file and added a comment 👍